cpp/ql/test/query-tests/Security/CWE/CWE-570/test.cpp

```diff
@@ -224,4 +224,12 @@ void good_new_catch_exception_in_conversion() {
   try {
     long* p = (long*) new int; // GOOD
   } catch(const std::bad_alloc&) { }
-}
\ No newline at end of file
+}
+
+// The 'n' parameter is just to distinquish it from the overload further up in this file.
+void *operator new(std::size_t, int n, const std::nothrow_t &);
+
+void test_operator_new_without_exception_spec() {
+  int* p = new(42, std::nothrow) int; // GOOD [FALSE POSITIVE]
```

Comment:
```suggestion
  int* p = new(42, std::nothrow) int; // GOOD
```
I assume this was fixed by the change.

Reply:
Oops! Indeed it was.